try:
    f=open('xxx','r')
except Exception as e:
    print('Erro: %s' % e)